Mail Scripts/Case details/Case details notification.js

```diff
@@ -0,0 +1,110 @@
+(function runMailScript(current, email, email_action, event) {
+    var gr = new GlideRecord('sn_customerservice_case'); // Replace with case table name)

```

Comment:
You already have current, why are you getting it again

Reply:
I thought it was necessary when calling a field. So, I can use getLink(), number, sys_id etc without using current.?

Reply:
Just simplified it.

Reply:
For future reference, that GlideRecord isn't necessary because that's what is provided through the current object. In doing this you add another call to the database only to get the same thing you already had.

Additionally, the variable name `gr` carries risks because it's used a lot in ServiceNow and unless you wrap your code in a function, it can collide with other declared instances of gr. While you are not at that specific risk here, that's a good practice to change